go/vt/vtgate/executor_test.go

```diff
@@ -1883,15 +1883,14 @@ func TestExecutorMaxPayloadSizeExceeded(t *testing.T) {
 	session := NewSafeSession(&vtgatepb.Session{TargetString: "@master"})
 	warningCount := warnings.Counts()["WarnPayloadSizeExceeded"]
 	testMaxPayloadSizeExceeded := []string{
-		"select * from main1",
 		"select * from main1",
 		"insert into main1(id) values (1), (2)",
 		"update main1 set id=1",
 		"delete from main1 where id=1",
 	}
 	for _, query := range testMaxPayloadSizeExceeded {
 		_, err := executor.Execute(context.Background(), "TestExecutorMaxPayloadSizeExceeded", session, query, nil)
-		if err == nil {
+		if err != nil {
```

Comment:
are we expecting a not-nil error? Then instead of `if` we should be doing `require.NotNil`

Reply:
Updated to use `require.NotNil`